test/mlir/affine_conversion/callback_ops.mlir

```diff
@@ -20,13 +36,13 @@ func @simple_softmax(%arg0: !ng.tensor<2x3xf32>, %arg1: !ng.tensor<1x!ng.i64>) -
 
 // Gemm Op
 // CHECK-LABEL: func @simple_gemm
-//       CHECK: %[[C1:.*]] = constant 0 : i64
-//       CHECK: %[[C2:.*]] = constant {{[0-9]+}} : i64
-//       CHECK: %0 = memref_cast %arg0 : memref<3x6xf32> to memref<*xf32>
-//       CHECK: %1 = memref_cast %arg1 : memref<6x4xf32> to memref<*xf32>
-//       CHECK: %2 = memref_cast %arg2 : memref<3x4xf32> to memref<*xf32>
-//       CHECK: %3 = memref_cast %arg3 : memref<3x4xf32> to memref<*xf32>
-//       CHECK: call @__mlir_callback_3_inputs(%0, %1, %2, %3, %[[C1]], %[[C2]]) : (memref<*xf32>, memref<*xf32>, memref<*xf32>, memref<*xf32>, i64, i64) -> ()
+//       CHECK: %0 = llvm.mlir.addressof @globalAttrs{{[0-9]+}} : !llvm<"{ i8, [3 x i64], [3 x i64], [3 x i64], [3 x i64] }*">
+//       CHECK: %[[C1:.*]] = constant {{[0-9]+}} : i64
+//       CHECK: %1 = memref_cast %arg0 : memref<3x6xf32> to memref<*xf32>
+//       CHECK: %2 = memref_cast %arg1 : memref<6x4xf32> to memref<*xf32>
```

Comment:
Please, use regular expresions to capture variable names. For example, here we can do:
```
%[[MC0:.*]] = memref_cast %arg0 : memref<3x6xf32> to memref<*xf32>
...
call @__mlir_callback_3_inputs(...%[[MC0]]...)
```